_includes/selector.html

```diff
@@ -503,7 +491,7 @@
                     add_pkgs_ln
                 ].filter(Boolean).join(" \\\n");
 
-                return lines;
+                return lines + " --solver=libmamba";
```

Comment:
I think the `--solver=libmamba` arg should go after `conda create` and before `-n rapids-23.08`.

Reply:
Functionally it doesn't need to, but if there's a style/hierarchy we should adhere to I'm happy to change. I just like that it's at the end so if someone has a problem with it, they can delete it easily.

Reply:
Updated to match the new rapids.ai ordering. https://github.com/rapidsai/rapids.ai/pull/328/commits/22e3bda7229a3f06d3df4a2c262dda73f717ff01